tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -377,6 +379,33 @@ def test_chunking_fast(self):
         self.assertEqual(output, [{"text": ANY(str)}])
         self.assertEqual(output[0]["text"][:6], "ZBT ZC")
 
+    @require_torch
```

Comment:
Heads up: this new test uses `speechbrain/m-ctc-t-large`, which is a 4.1 GB model.

I'm not familiar with the testing infrastructure, so if that's undesirable there may be other options for testing. However, there is no readily available official/popular "smaller" version of this model.

Reply:
This should be fine thanks :-)

Reply:
Shouldn't that be marked as `@slow` test ?

Reply:
Yes definitely indeed!